isthmus/src/main/java/io/substrait/isthmus/SubstraitRelVisitor.java

```diff
@@ -283,22 +284,30 @@ public Rel visit(org.apache.calcite.rel.core.Match match) {
 
   @Override
   public Rel visit(org.apache.calcite.rel.core.Sort sort) {
-    var input = apply(sort.getInput());
-    var fields =
-        sort.getCollation().getFieldCollations().stream()
-            .map(t -> toSortField(t, input.getRecordType()))
-            .collect(java.util.stream.Collectors.toList());
-    var convertedSort = Sort.builder().addAllSortFields(fields).input(input).build();
-    if (sort.fetch == null && sort.offset == null) {
-      return convertedSort;
+    Rel input = apply(sort.getInput());
+    Rel output = input;
+
+    // SortRel is added BEFORE FetchRel, if present
```

Comment:
Before can be a bit confusing -- perhaps make it clearer that one is operating on the other?

Reply:
This is a bit ambiguous for sure. Added a bit more language around this.